nilearn/datasets/tests/test_testing.py

```diff
@@ -157,6 +157,7 @@ def test_sender_bad_input(request_mocker):
 
 
 def test_dict_to_archive(tmp_path):
+    tmp_path = Path(str(tmp_path))
```

Comment:
This is a question.
tmp_path is already Path, why do we need to convert it to str and then back?